dateparser/custom_language_detection/fasttext.py

```diff
@@ -0,0 +1,46 @@
+import os
+
+import fasttext
+
+from dateparser_cli.fasttext_manager import fasttext_downloader
+from dateparser_cli.utils import date_parser_model_home, check_data_model_home_existance
+
+
+_supported_models = ["large.bin", "small.bin"]
+
+
+class FastTextCache:
+    model = None
+
+
+def _load_fasttext_model():
+
+    if FastTextCache.model:
+        return FastTextCache.model
+
+    check_data_model_home_existance()
+    model_path = None
+    downloaded_models = os.listdir(date_parser_model_home)
+
+    for downloaded_model in downloaded_models:
+        if downloaded_model in _supported_models:
+            model_path = date_parser_model_home + "/" + downloaded_model
```

Comment:
This is not windows-compatible

Reply:
Done 👍🏻 & it's now windows compatible but tox/setup.py does not seem to be working on windows.